### PR TITLE
Added param description service for discharge measurements/error and now

### PR DESCRIPTION
**Pull request by thongsav-usgs**
plotting data in UI.

### CODE REVIEW

**Comment by mhines-usgs** [comment]
pulling this work locally and will test it out against dev.

**Comment by mhines-usgs** [comment]
looks ok to me, perhaps @ee-usgs has more feedback for you before merging.

File: gcmrc-services/src/main/java/gov/usgs/cida/gcmrcservices/column/ColumnResolver.java [hunk 1]
```diff
@@ -33,7 +33,7 @@
 	private static ColumnResolver resolver = null;
 	protected Map<String, ColumnMetadata> CM_LOOKUP;
 	protected Map<String, ColumnMetadata> qwColumnMetadatas;
-	protected Map<String, ColumnMetadata> qwErrorColumnMetadatas;
+	protected Map<String, ColumnMetadata> dischargeErrorColumnMetadatas;
```
Comment (ee-usgs) [inline gcmrc-services/src/main/java/gov/usgs/cida/gcmrcservices/column/ColumnResolver.java]: Great!  If this really is just applicable to discharge error data, then the updated naming is much better.

File: gcmrc-services/src/main/java/gov/usgs/cida/gcmrcservices/jsl/station/StationDischargeErrorSpec.java [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+import gov.usgs.webservices.jdbc.spec.mapping.SearchMapping;
+import gov.usgs.webservices.jdbc.spec.mapping.WhereClauseType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * @author thongsav
+ */
+public class StationDischargeErrorSpec extends GCMRCSpec {
```
Comment (ee-usgs) [inline gcmrc-services/src/main/java/gov/usgs/cida/gcmrcservices/jsl/station/StationDischargeErrorSpec.java]: Wait, how is this different than DischargeErrorSpec?  I don’t see that one deleted if its a rename.

File: gcmrc-ui/src/main/webapp/stationview.jsp [hunk 3]
```diff
@@ -126,6 +126,7 @@
 		<script src="${relativePath}services/service/param/jsonp/param?site=${stationName}&jsonp_callback=GCMRC.Page.paramsLoad"></script>
 		<script src="${relativePath}services/service/param/jsonp/bs?site=${stationName}&jsonp_callback=GCMRC.Page.bsLoad"></script>
 		<script src="${relativePath}services/service/param/jsonp/qw?site=${stationName}&jsonp_callback=GCMRC.Page.qwLoad&orderby=displayOrder, sampleMethod desc"></script>
+		<script src="${relativePath}services/service/param/jsonp/dischargeError?site=${stationName}&jsonp_callback=GCMRC.Page.qwLoad&orderby=displayOrder, sampleMethod desc"></script>
```
Comment (ee-usgs) [inline gcmrc-ui/src/main/webapp/stationview.jsp]: Wow, this seems like really minimal changes to add error bars.  What is it that specifies the rendering as error bars in the chart?  Is it in the JSON?

So, I assume that this script tag returns JSON of the error bar data coming into the page, parameterized by the station to load just the correct data.  Don’t we need more filtering here, like a time range or something?  Or do we just always load all the station error data to the UI.  I see that this is the pattern, but that has the potential to be really large.

Comment (thongsav-usgs) [inline gcmrc-ui/src/main/webapp/stationview.jsp]: This actually only returns Metadata about the parameters that might be displayed. Actually data gets requested when you hit build. All the UI code works as long as it is returned in the expected format and the Metadata is preloaded, which is what this line is for.

File: gcmrc-services/src/main/java/gov/usgs/cida/gcmrcservices/jsl/station/StationDischargeErrorSpec.java (hunk 2 above)
Comment (thongsav-usgs) [inline gcmrc-services/src/main/java/gov/usgs/cida/gcmrcservices/jsl/station/StationDischargeErrorSpec.java]: This creates Metadata for the UI which describes the error data coming back, pretty much type identifiers.

**Comment by mhines-usgs** [comment]
Is this OK to merge then?

**Pull request merged**